Kickstarter-iOS/Features/RewardsCollection/Controller/RewardsCollectionViewController.swift

```diff
@@ -203,6 +236,13 @@ final class RewardsCollectionViewController: UICollectionViewController {
       self.collectionView.topAnchor.constraint(equalTo: self.view.topAnchor)
     ])
 
+    if featurePostCampaignPledgeEnabled() {
+      self.headerView.leftAnchor.constraint(equalTo: self.view.leftAnchor).isActive = true
+      self.headerView.rightAnchor.constraint(equalTo: self.view.rightAnchor).isActive = true
+      self.headerView.topAnchor.constraint(equalTo: self.view.topAnchor).isActive = true
+      self.headerView.widthAnchor.constraint(equalTo: self.view.widthAnchor).isActive = true
```

Comment:
nit: Does the width anchor do anything when we already have left and right? Also, the height is underconstrained - does it give a build warning? If it does, can you add a height constraint and just set it to the intrinsic content size?

Reply:
I was seeing issues originally, but it looks like now that I'm setting the collectionView's topAnchor directly, that issue has been fixed 🤷 .